README.md

```diff
@@ -77,18 +69,19 @@ See [related documentation](./guides/alerting/alerting.md) for more information.
 
 # Installation
 
-_Trento_ is intended to run in many ways, depending on needs, and also the different components have their own requirements and constraints.
+_Trento_ can be installed in various ways, depending on your needs and the requirements of its different components. Please refer to the specific [Agent](https://github.com/trento-project/agent) documentation for more information.
+
+## Installing Trento on a k8s Cluster
 
-Please check the specific [Agent](https://github.com/trento-project/agent) and [Runner](https://github.com/trento-project/runner) documentation for respective details.
+For detailed instructions on installing Trento on a k8s cluster, please refer to our [helm-charts](https://github.com/trento-project/helm-charts) repository.
 
-If you intend to install Trento on a k8s cluster, you may be interested in our [helm-charts](https://github.com/trento-project/helm-charts) repo for detailed instructions.
+## Installing Trento Web locally for Development
 
-If otherwise you want to play around with the current repository, go ahead, [hack on the trento](./guides/development/hack_on_the_trento.md) and **have a lot of fun**!
+To install Trento Web locally for development, please refer to our [hack on the Trento Web](./guides/development/hack_on_the_trento.md) guide for detailed instructions.
 
 # Support
 
-Please only report bugs via [GitHub issues](https://github.com/trento-project/web/issues);
-for any other inquiry or topic use [GitHub discussion](https://github.com/trento-project/trento/discussions).
+Please only report bugs via [GitHub issues](https://github.com/trento-project/web/issues) and for any other inquiry or topic, use [GitHub discussion](https://github.com/trento-project/trento/discussions).
```

Comment:
https://github.com/trento-project/trento/discussions is referring to the legacy trento app.

I'd remove that reference, and if we decide to enable discussions for web as well, we can add the proper link when needed.

Reply:
Yeah makes sense thanks :D